verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -64,10 +64,8 @@ VNameRef ConvertToVnameRef(const VName& vname,
 
 // Prints Kythe facts in proto format to stdout.
 void PrintKytheFactsProtoEntries(const IndexingFactNode& file_list_facts_tree) {
-  KytheFactsExtractor kythe_extractor(
-      GetFileListDirFromRoot(file_list_facts_tree),
-      /*previous_files_scopes=*/nullptr);
-  auto indexing_data = kythe_extractor.ExtractKytheFacts(file_list_facts_tree);
+  auto indexing_data =
+      KytheFactsExtractor::ExtractFileList(file_list_facts_tree);
```

Comment:
Can you keep the name `ExtractKytheFacts`? `ExtractFileList` being the entry point is a bit confusing. `ExtractKytheFacts` is what the users want.